src/visualization/types.jl

```diff
@@ -4,16 +4,29 @@
 # TimeIntegratorSolution.
 #
 # Note: This is an experimental feature and may be changed in future releases without notice.
-const TrixiODESolution = Union{ODESolution{T, N, uType, uType2, DType, tType, rateType, P} where
-    {T, N, uType, uType2, DType, tType, rateType, P<:ODEProblem{uType_, tType_, isinplace, P_, F_} where
-     {uType_, tType_, isinplace, P_<:AbstractSemidiscretization, F_}}, TimeIntegratorSolution}
+const TrixiODESolution = Union{
+                               ODESolution{T, N, uType, uType2, DType, tType, rateType, P
+                                           } where
+                                   {
+                                   T,
+                                   N,
+                                   uType,
+                                   uType2,
+                                   DType,
+                                   tType,
+                                   rateType,
+                                   P <: ODEProblem{uType_, tType_, isinplace, P_, F_
+                                              } where
+                                       {uType_, tType_, isinplace,
+                                        P_ <: AbstractSemidiscretization, F_},
+                               }, TimeIntegratorSolution}
```

Comment:
```suggestion
const TrixiODESolution = Union{ODESolution{T, N, uType, uType2,
                                           DType, tType, rateType, P
                                           } where {T, N, uType, uType2,
                                                    DType, tType, rateType,
                                                    P <: ODEProblem{uType_, tType_, 
                                                                    isinplace, P_, F_
                                                                    } where {uType_, tType_, isinplace,
                                                                             P_ <: AbstractSemidiscretization, F_},
                               }, TimeIntegratorSolution}
```
Idk, something like that might work.

Reply:
This one is tricky and there's not much that works. 3881f3ff3 might be an improvement, but if it is, it is really minuscule.

Reply:
I'm a big fan of `#! format: off` for this one.

Reply:
👍

Reply:
Restored to previous format in a820303fc